sklearn/model_selection/_split.py

```diff
@@ -664,14 +664,17 @@ class TimeSeriesSplit(_BaseKFold):
     n_splits : int, default=3
         Number of splits. Must be at least 1.
 
+    max_train_size : int, optional
+        Maximum size for a single training fold.
```

Comment:
I'm never sure about the correct use of "fold". Let's call it a "training set" or a "training sample".